Image_Msg img1

---
float32[] pred